tests/vhost/test-vhost.c

```diff
@@ -218,22 +218,22 @@ static void test_vhost_multivm_(int qemu_exit_signal)
 	g_assert(!error);
 
 	/* instanciate brick */
-	vhost_00 = pg_vhost_new("vhost-00", PG_VHOST_USER_DEQUEUE_ZERO_COPY,
+	vhost_00 = pg_vhost_new("vhost-00", 0,
 				&error);
 	g_assert(!error);
 	g_assert(vhost_00);
 
-	vhost_01 = pg_vhost_new("vhost-01", PG_VHOST_USER_DEQUEUE_ZERO_COPY,
+	vhost_01 = pg_vhost_new("vhost-01", 0,
 				&error);
 	g_assert(!error);
 	g_assert(vhost_01);
 
-	vhost_10 = pg_vhost_new("vhost-10", PG_VHOST_USER_DEQUEUE_ZERO_COPY,
+	vhost_10 = pg_vhost_new("vhost-10", 0,
 				&error);
 	g_assert(!error);
 	g_assert(vhost_10);
 
-	vhost_11 = pg_vhost_new("vhost-11", PG_VHOST_USER_DEQUEUE_ZERO_COPY,
+	vhost_11 = pg_vhost_new("vhost-11", 0,
```

Comment:
you removed PG_VHOST_USER_DEQUEUE_ZERO_COPY from tests but not really from vhost's API